### DISCUSSION TITLE
Kardiachain (EVM compatible) support

### DISCUSSION THREAD

**Discussion by Lan-Phan** [Web3Auth/Web3Auth#862] [Feature Requests]
Hello,

We're from KardiachainLabs and we'd love if web3auth can add our Kardiachain to your product.
Here is information about our chain:

   Network Name: KardiaChain Mainnet
   New RPC URL: https://rpc.kardiachain.io
   Chain ID: 24
   Block Explorer URL: https://explorer.kardiachain.io/
   Current Symbol: KAI

More information is here: https://www.kardiachain.io/

Thanks.

**Comment by himanshuchawla009** [comment]
You want us to add in our docs?

**Reply by Lan-Phan** [reply]
nope, just because we cannot use your wallet with our Kardiachain.
From this https://web3auth.io/docs/sdk/web/providers/evm , we found that your supported chain list is from chainlist.org, my new question is how long do you update supported chain list from chainlist.org? We're going to add Kardiachain there.

**Reply by himanshuchawla009** [reply]
@shahbaz17

**Reply by shahbaz17** [reply]
Hey @Lan-Phan 

That's just for reference, let us update that to include all EVM chains.

Meanwhile, do you want us to add this chain to the docs? For devs to easily integrate.

**Comment by Lan-Phan** [comment]
Hi @himanshuchawla009 @shahbaz17 ,

Our PR to update chain id 24 from Dithereum to our Kardiachain just been merged, and now it's available on chainlist already. Please help to update your SDK that chain id 24 is our Kardiachain, thanks so much.
